github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -340,6 +340,8 @@ presubmits:
           env:
           - name: GOOGLE_APPLICATION_CREDENTIALS
             value: /etc/gcs/service-account.json
+          - name: GITHUB_TOKEN
```

Comment:
Would it make sense to simply inject the preset in this job?

Reply:
This was the only instance of the env variable GITHUB_TOKEN that was set in the prowjobs - this env variable being set in the preset caused an issue in another job - [push-release-kubevirt-tag](https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/push-release-kubevirt-tag/1491436583453200384).

As this env variable was only used in one job - there was less value including it in the preset.

Reply:
Perfect then!